draft-ietf-quic-qpack.md

```diff
@@ -1551,6 +1551,128 @@ the smallest number of bytes.
 | 97    | x-frame-options                  | deny                                                        |
 | 98    | x-frame-options                  | sameorigin                                                  |
 
+
+# Encoding and Decoding Examples
+
+The following examples represent a series of exchanges between an encoder and a
+decoder.  The state of the encoder's dyanmic table is shown, along with its
+current size.  Each entry is shown with the Absolute Index of the entry (Abs),
+the current number of outstanding encoded field sections with references to that
+entry (Ref), along with the name and value.  Entries above the 'acknowledged'
+line have been acknowledged by the decoder.
+
+## Literal Field Line With Name Reference
+
+~~~
+Data                | Interpretation
+                              | Encoder's Dynamic Table
+
+Stream: 0
+0000                | Required Insert Count = 0, Base = 0
+510b 2f69 6e64 6578 | :path=/index.html
+2e68 746d 6c
+
+                                Abs Ref Name        Value
+                                ^-- acknowledged --^
+                                Size=0
+~~~
+
+## Dynamic Table
+
+~~~
+Stream: Encoder
+3f8b                | Set Table Capacity=170
+01c0 0c77 7777 2e69 | Insert with static name  reference
+6574 662e 6f72 67   |  (:authortiy=www.ietf.org)
+
+                                Abs Ref Name        Value
+                                ^-- acknowledged --^
+                                 1   0  :authority  www.ietf.org
+                                Size=54
+
+Stream: 4
+0280                | Required Insert Count = 1, Base = 0
+10                  | Indexed field line with post-base index
```

Comment:
Here and below for Stream 8: in addition to naming the representation type, consider writing out the encoded header name and value for clarity.